### ISSUE TITLE
Can't install package due to HTTP 429 errors

### ISSUE THREAD

**Issue by Kaishiyoku** [simonhamp/gitamic-support#1]
I tried deploying my application and thus installing all needed Composer packages and every time I'm getting a HTTP 429 error from the private repository.

```
 [Composer\Downloader\TransportException]
  The "[...]/gitamic-2.2.2.zip" file could not be downloaded (HTTP/2 429 )
```

I don't know the rate limiting configuration of the private repository but I already waited around two hours and I'm still getting 429 errors.

Sincere regads.

**Comment by simonhamp** [comment]
Thanks for reporting this @Kaishiyoku 

I'll look into it shortly

**Comment by simonhamp** [comment]
@Kaishiyoku have you added your fingerprint to your license key in your `auth.json`?

**Comment by Kaishiyoku** [comment]
Yes I did. I also tried installing manually with authentication prompt.

**Comment by simonhamp** [comment]
Thanks for confirming that. Can I just check a few more things?
- what version of Composer are you running on the server you're deploying from?
- are you getting this 429 from your local machine too or only from your server?

**Comment by Kaishiyoku** [comment]
I re-checked. Installing on my local machine works; I had added the credentials to my global auth.json file. I took a look inside my local auth.json and I saw my mistake...I didn't add :{domain-name} when entering the password. So everything's okay :) Got it working now. I guess I should have slept over it; it was quite late yesterday :D

Anyway, thank you very much for the quick response, appreciate it! Have a wonderful day.
